### DISCUSSION TITLE
Some functionalities no longer working after deploying to production.

### DISCUSSION THREAD

**Discussion by zwouter** [tailwindlabs/tailwindcss#8974] [Help]
Hi, I'm working on a Laravel project with Tailwindcss for my styling. A few days ago I deployed the system to an app platform in DigitalOcean, and after doing so, a lot of tailwind's functionality no longer works. 
For example, I have some date input fields to which I added `hover:cursor-text` to improve their styling. On my local host this works as expected, but it does not seem to work in production.
Could I have made a mistake while deploying it, or are there just some options that don't work in production? I can't find anybody with a similar problem online, so any help is appreciated!

**Comment by wongjn** [comment]
Have a look at the CSS generated between local and production, perhaps the differences between them could give you a clue of where it's gone wrong?
